media-tv/plex-media-server/plex-media-server-1.14.0.5470.ebuild

```diff
@@ -0,0 +1,103 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit systemd unpacker user
+
+DESCRIPTION="A free media library that is intended for use with a plex client"
+HOMEPAGE="https://www.plex.tv/"
+COMMIT="9d51fdfaa"
+
+_APPNAME="plexmediaserver"
+_USERNAME="plex"
+_SHORTNAME="${_USERNAME}"
+_FULL_VERSION="${PV}-${COMMIT}"
+
+SRC_URI="
+	amd64? ( https://downloads.plex.tv/${PN}/${_FULL_VERSION}/${_APPNAME}_${_FULL_VERSION}_amd64.deb )
+	x86? ( https://downloads.plex.tv/${PN}/${_FULL_VERSION}/${_APPNAME}_${_FULL_VERSION}_i386.deb )"
+
+LICENSE="Plex"
+SLOT="0"
+KEYWORDS="-* ~amd64 ~x86"
+RESTRICT="mirror bindist strip"
+IUSE="avahi system-openssl"
+
+DEPEND="sys-apps/fix-gnustack"
+RDEPEND="avahi? ( net-dns/avahi )
+	system-openssl? ( dev-libs/openssl:0 )"
+
+QA_DESKTOP_FILE="usr/share/applications/plexmediamanager.desktop"
+QA_PREBUILT="*"
+QA_MULTILIB_PATHS=( "usr/lib/${_APPNAME}/.*" )
+
+S="${WORKDIR}"
+
+PATCHES=( "${FILESDIR}/plexmediamanager.desktop.patch" )
+
+pkg_setup() {
+	enewgroup ${_USERNAME}
+	enewuser ${_USERNAME} -1 /bin/bash /var/lib/${_APPNAME} "${_USERNAME},video"
+}
+
+src_unpack() {
+	unpack_deb ${A}
+}
+
+src_install() {
+	# Move the config to the correct place
+	local config_vanilla="/etc/default/${_APPNAME}"
+	local config_path="/etc/${_SHORTNAME}"
+	dodir "${config_path}"
+	insinto "${config_path}"
+	doins "${config_vanilla#/}"
+	sed -e "s#${config_vanilla}#${config_path}/$(basename "${config_vanilla}")#g" \
+		-i "${S}"/usr/sbin/start_pms || die
+	# Remove Debian specific files
+	rm -r "usr/share/doc" || die
+	# Remove buggy openssl library
+	if use system-openssl; then
+		rm -f usr/lib/plexmediaserver/libssl.so.1.0.0 || die
+	fi
+	# Copy main files over to image and preserve permissions so it is portable
+	cp -rp usr/ "${ED}" || die
+	# Make sure the logging directory is created
+	local logging_dir="/var/log/pms"
+	keepdir "${logging_dir}"
+	fowners "${_USERNAME}":"${_USERNAME}" "${logging_dir}"
+	# Create default library folder with correct permissions
+	local default_library_dir="/var/lib/${_APPNAME}"
+	keepdir "${default_library_dir}"
+	fowners "${_USERNAME}":"${_USERNAME}" "${default_library_dir}"
+	# Install the OpenRC init/conf files depending on avahi.
+	if use avahi; then
+		doinitd "${FILESDIR}/init.d/${PN}"
+	else
+		cp "${FILESDIR}/init.d/${PN}" "${S}/${PN}" || die
+		sed -e '/depend/ s/^#*/#/' \
+		-e '/need/ s/^#*/#/' \
+		-e '1,/^}/s/^}/#}/' -i "${S}/${PN}" || die
+		doinitd "${S}/${PN}"
```

Comment:
see above. copy to `${T}` and install from there.

Reply:
Fixed, thanks.